packages/cli/src/commands/certs/ls.ts

```diff
@@ -38,10 +40,20 @@ async function ls(
     return 1;
   }
 
+  if (
+    typeof nextTimestamp !== undefined &&
+    (Number.isNaN(limit) || limit! > 100)
```

Comment:
We try to avoid using `!` when possible:

```suggestion
    typeof nextTimestamp !== undefined &&
    typeof limit === 'number' &&
    (Number.isNaN(limit) || limit > 100)
```

Reply:
Thanks 🙇🏾  there's a few of those, I'll update across the whole PR.